typescript/packages/subsurface-viewer/src/layers/wells/wellsLayer.ts

```diff
@@ -254,33 +267,39 @@ export function getSize(
 
 export default class WellsLayer extends CompositeLayer<WellsLayerProps> {
     initializeState(): void {
-        let data = this.props.data as unknown as FeatureCollection;
-        if (typeof data !== "undefined" && !isEqual(data, [])) {
-            if (this.props.ZIncreasingDownwards) {
-                data = invertPath(data);
-            }
+        let data = this.props.data as FeatureCollection<GeometryCollection>;
+        const refine = this.props.refine;
 
-            checkWells(data);
+        if (typeof data === "undefined" || isEqual(data, [])) {
```

Comment:
isn't `data == undefined` or `! data` good enough ?
I'm always wondering the added value of `typeof data === "undefined"` ...

Reply:
I agree. Fixed.